systemd/fmn-backend@.service

```diff
@@ -1,13 +1,11 @@
 [Unit]
-Description=FMN.consumer backend #%i
+Description=FMN Delivery Service #%i
 After=network.target
-Documentation=https://github.com/fedora-infra/fmn.consumer/
+Documentation=https://github.com/fedora-infra/fmn/
 
 [Service]
-ExecStart=/usr/bin/python2 /usr/lib/python2.7/site-packages/fmn/consumer/backend.py
+ExecStart=/usr/bin/twistd -n -l - -y /usr/lib/share/fmn/delivery_service.tac
 Type=simple
-User=root
-Group=root
```

Comment:
What user will this run as with these removed? I presume root is the default?

Reply:
Possibly, but it doesn't need to run as root so I thought it was better to not require that.